api/data/tacoma_db.go

```diff
@@ -11,11 +11,22 @@ import (
 
 // TacomaOfficer is the object model for Tacoma PD officers
 type TacomaOfficer struct {
+	Date       string `json:"date,omitempty"`
+	FirstName  string `json:"first_name,omitempty"`
+	LastName   string `json:"last_name,omitempty"`
+	Title      string `json:"title,omitempty"`
+	Department string `json:"department,omitempty"`
+	Salary     string `json:"salary,omitempty"`
+}
+
+// tacomaOfficer is an internal intermediary between the returned SQL rows data
+// and the actual JSON return itself.
+type tacomaOfficer struct {
 	Date       time.Time    `json:"date,omitempty"`
-	FirstName  string       `json:"first_name,omitempty"`
-	LastName   string       `json:"last_name,omitempty"`
-	Title      string       `json:"title,omitempty"`
-	Department string       `json:"department,omitempty"`
+	FirstName  nulls.String `json:"first_name,omitempty"`
+	LastName   nulls.String `json:"last_name,omitempty"`
+	Title      nulls.String `json:"title,omitempty"`
+	Department nulls.String `json:"department,omitempty"`
 	Salary     nulls.String `json:"salary,omitempty"`
 }
```

Comment:
Why do both the Tacoma officer structs have the `json` bits but not both the Seattle officer structs?

Reply:
Whoop because I'm bad at consistency haha.

Reply:
i _think_ it was just copy/paste. Since we're never marshalling little t `tacomaOfficer`, I don't think we need the `json` tags